protostar/commands/call_command.py

```diff
@@ -58,41 +79,38 @@ def arguments(self):
         ]
 
     async def run(self, args: Any):
+        msg = OutputCommandUtil(args).get_messenger(self._log_color_provider)
```

Comment:
please give variables more meaningful names,  - `messenger()` would be nicer here

Reply:
`write`? I would like to keep this a verb

Reply:
done